src/common/containers/repository-setup.js

```diff
@@ -5,39 +5,40 @@ import { withRouter } from 'react-router';
 
```

Comment:
`createSnap` goes through SSO redirect and we want to create webhooks and request builds after this happens (to be sure whole process succeeded). That's why /owner/name/setup route is needed to finalise setting up repository before redirecting to builds view.